From: Boyuan Yang <byang@debian.org>
Date: Wed, 13 Sep 2023 12:46:11 -0400
Subject: Use a local copy of objects.inv

Use a local copy of objects.inv from
* doc.python.org
* cairographics.org
* cffi.readthedocs.io

...instead of downloading them each time from the internet
---
 docs/conf.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 06ff6a0..869fca4 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -80,7 +80,7 @@ texinfo_documents = [(
 ]
 
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/3/', None),
-    'pycairo': ('https://cairographics.org/documentation/pycairo/2/', None),
-    'cffi': ('https://cffi.readthedocs.io/en/latest/', None),
+    'python': ('https://docs.python.org/3/', '../debian/python.org_objects.inv'),
+    'pycairo': ('https://cairographics.org/documentation/pycairo/2/', '../debian/cairographics.org_pycairo2_objects.inv'),
+    'cffi': ('https://cffi.readthedocs.io/en/latest/', '../debian/cffi.readthedocs.io_cffi_objects.inv'),
 }
